Singular/dyn_modules/systhreads/libthread.h

```diff
@@ -24,7 +24,7 @@ namespace LibThread {
   // Job scheduling
   Job *startJob(ThreadPool *pool, Job *job, leftv arg);
   Job *startJob(ThreadPool *pool, Job *job);
-  Job *scheduleJob(ThreadPool *pool, Job *job, long ndeps, Job **deps);
+//   Job *scheduleJob(ThreadPool *pool, Job *job, long ndeps, Job **deps);
```

Comment:
grep didn't find any caller for this (@rbehrends or did I miss something?)

Reply:
This exists as a C++ API version of the Singular function `scheduleJob()`, which is implemented independently. It is not called by the implementation of the Singular function, because validation and error checking is interleaved with the actual logic. This is some unfortunate code duplication that should be cleaned up at some point.